src/main/webapp/app/tasks/task-form/task-form.component.scss

```diff
@@ -10,3 +10,8 @@ form {
 .mat-icon {
   cursor: pointer;
 }
+
+.errorDiv {
```

Comment:
best practice for css classes is kebab case. Additionally i can't find any usages of this class.

Reply:
I had added this class for testing purpose , It should be removed and regarding the kebab case yes it is indeed a best practice